### PR TITLE
Annotated several conversations

### PR DESCRIPTION
**Pull request by ArjaanBuijk**
I did all the annotations with Rasa X of website-demo, except the addition of synonym `custom actions`
Note that Rasa X re-arranged some synonyms compared to master, without any content change.

### CODE REVIEW

File: data/nlu/nlu.md [hunk 1]
```diff
@@ -3045,6 +3046,7 @@
 - we think 4 million INR/ year
 - we're building a conversational assistant for our employees to book meeting rooms.
 - £50k
+- botonic
```
Comment (erohmensing) [inline data/nlu/nlu.md]: is this in the context of a user giving their company? if so we should annotate it as such

File: data/nlu/nlu.md [hunk 2]
```diff
@@ -4846,6 +4859,11 @@
 - Core
 - CORE
 
+## synonym:custom actions
+- actions
+- Custom actions
+- Actions
```
Comment (erohmensing) [inline data/nlu/nlu.md]: Agreed this is a good idea bc it'll improve greg's search. did you add this?

File: data/nlu/nlu.md (hunk 1 above)
Comment (erohmensing) [reply data/nlu/nlu.md]: (same with wordpress and slack, actually)

Comment (ArjaanBuijk) [reply data/nlu/nlu.md]: Both were in the context of a user providing information related to switching from another platform. The story expects the `enter_data` intent

File: data/nlu/nlu.md (hunk 2 above)
Comment (ArjaanBuijk) [reply data/nlu/nlu.md]: Yes, I indeed added this with Greg's search update in mind. I did not yet test with Greg's search, but plan to do so once that one is merged.

**Pull request merged**